### PR TITLE
Move cell sets to s3

### PR DESCRIPTION
**Pull request by cosa65**
When uploading relevant data to the aws services, upload cellSets object to a particular s3 table cell-sets-{CLUSTER_ENV} instead of adding it to the experiments data in dynamodb.

This probably has things in common with https://github.com/biomage-ltd/data-ingest/pull/19/files, both have the cluster_env flag added.

### CODE REVIEW

**Review by JuanluOnieva-biomage — APPROVED** [review]

File: src/5_Upload-to-aws.py [hunk 1]
```diff
@@ -223,29 +248,33 @@ def main():
...
     samples_data = json.loads(json.dumps(samples_data), parse_float=Decimal)
-    
+
     access_key = os.getenv("AWS_ACCESS_KEY_ID")
     secret_access_key = os.getenv("AWS_SECRET_ACCESS_KEY")
 
+    experiments_key = f"experiments-{ENVIRONMENT}"
+    samples_key = f"samples-{ENVIRONMENT}"
+    cell_sets_bucket = f"cell-sets-{ENVIRONMENT}"
+    r_object_bucket, r_object_key = FILE_NAME.split("/", 1)
```
Comment (alexvpickering) [inline src/5_Upload-to-aws.py:266]:
```suggestion
    experiments_key = f"experiments-{CLUSTER_ENV}"
    samples_key = f"samples-{CLUSTER_ENV}"
    cell_sets_bucket = f"cell-sets-{CLUSTER_ENV}"
    r_object_bucket, r_object_key = FILE_NAME.split("/", 1)
```

File: src/5_Upload-to-aws.py [hunk 2]
```diff
@@ -223,29 +248,33 @@ def main():
...
 
-    print("uploading to dynamodb experiments table...")
+    print(f"uploading to dynamodb experiments table {experiments_key}...")
     dynamo = boto3.resource(
         "dynamodb",
         aws_access_key_id=access_key,
         aws_secret_access_key=secret_access_key,
         region_name="eu-west-1",
     ).Table(f"experiments-{CLUSTER_ENV}")
```
Comment (alexvpickering) [inline src/5_Upload-to-aws.py:274]:
```suggestion
    ).Table(experiments_key)
```

File: src/5_Upload-to-aws.py [hunk 3]
```diff
@@ -254,19 +283,26 @@ def main():
     ).Table(f"samples-{CLUSTER_ENV}")
```
Comment (alexvpickering) [inline src/5_Upload-to-aws.py:283]:
```suggestion
    ).Table(samples_key)
```

**Review by alexvpickering — CHANGES_REQUESTED** [review]
See suggestions. I'll also run with some user data to test

**Review by alexvpickering — APPROVED** [review]

**Pull request merged**